extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -102,6 +103,13 @@ const ALLOWED_AD_TYPES = map({
   'doubleclick': true,
 });
 
+/** @enum {boolean} */
+const DISALLOWED_AD_ATTRS = {
```

Comment:
I believe this has been discussed before, but could I know why we choose the blacklist instead of whitelist. 
Asking because we are using whitelist for validator check, and it is more stricter I believe.

Reply:
this component can be used by other ad networks. we just blindly copy over attributes that are needed by the particular ad networks.

those attributes are black listed because we do not allow arbitrary layout and dimension for story ad slots

Reply:
Talked with @lannka offline. We agreed to go with whitelist. Created #20003 to follow up.

Reply:
Sounds good.